modules/gitops-release-notes-1-13-2.adoc

```diff
@@ -0,0 +1,70 @@
+
+[id="gitops-release-notes-1-13-2_{context}"]
+= Release notes for {gitops-title} 1.13.2
+
+{gitops-title} 1.13.2 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, and 4.16.
+
+[id="errata-updates-1-13.2_{context}"]
+== Errata updates
+
+[id="rhsa-2024:8581-gitops-1-13-2-security-update-advisory_{context}"]
+=== RHSA-2024:8581 - {gitops-title} 1.13.2 security update advisory
+
+Issued: 2024-10-29
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:8581[RHSA-2024:8581]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-13-2_{context}"]
+== Fixed issues
+
+* Before this update, Argo CD could not obtain the appropriate transport layer security (TLS) certificate for Helm Open Container Initiative (OCI) registries when the URL included a path or port number. With this update, a fix is introduced in upstream Argo CD to correctly parse the URL and return a valid certificate. link:https://issues.redhat.com/browse/GITOPS-5081[GITOPS-5081]
+
+* Before this update, you could not log in to the Argo CD web console UI after a Red Hat OpenShift Service on an Amazon Web Services (AWS) cluster, that includes a {gitops-shortname} Operator and an Argo CD instance configured with Dex-based SSO, resumed from hibernation. The login screen would display an error indicating an invalid redirect Uniform Resource Identifier (URI) in the Dex configuration. This update fixes the issue by ensuring that the correct Dex redirect URL is updated in the Argo CD configuration whenever the Argo CD server route is modified. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
+
+[id="breaking-change-1-13-2_{context}"]
+== Breaking change
+
+[id="additional-configurations-for-cluster-scoped-rollouts-instance-installation_{context}"]
+=== Additional configurations for cluster-scoped rollouts instance installation
+
+* When you upgrade to {gitops-title} v1.13, if you want to create the cluster-scoped rollouts installation outside the default installation namespace, `openshift-gitops`, you must host it in the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable of the `Subscription` resource. The {gitops-title} Operator does not support cluster-scoped rollouts installation if the namespace is not defined in the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable.
++
+In previous versions of {gitops-title}, Argo Rollouts used the `NAMESPACE_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable of the `Subscription` resource to check if the cluster-scoped rollouts instance can host a cluster-scoped rollouts installation in the user-defined namespace. link:https://issues.redhat.com/browse/GITOPS-5640[GITOPS-5640]
```

Comment:
The Prow build is reporting:

> ERROR: Do not include links to JIRA issues that require a login (https://issues.redhat.com/browse/GITOPS-5640). I see that the Jiras has a security level set: 

![Screenshot from 2024-10-31 21-19-13](https://github.com/user-attachments/assets/f4eb555f-121c-4c9f-be7a-d6b3c209ba0a)

The engineer might be able to set the Jira as public but if any customer-specific information exists, it will need to be removed.

Reply:
Replied in the previous comment.